CHANGELOG.rst

```diff
@@ -365,3 +365,14 @@ New Modules
 - fact_diff - Find the difference between currently set facts
 - update_fact - Update currently set facts
 - validate - Validate data with provided criteria
+
+v4.0.0
+======
+
+Major Changes
+-------------
+
+- This release mainly fixes the breaking changes in the `netaddr` library.
+- With the new release of `netaddr` 1.0.0, the `IPAddress.is_private()` method has been removed and instead, the `IPAddress.is_global()` method has been extended to support the same functionality. This change has been reflected in the `ipaddr` filter plugin.
+- Bumping `netaddr` to `>=0.10.1`, means that starting from this release, the minimum `netaddr` version this collection requires is `>=0.10.1`.
+- Removes compat files as we stopped testing against python 2.7.
```

Comment:
test related updates do not need to be part of changelog.

Reply:
cool, removing them